decidim/decidim
# Fix components navbar in consultations mobile

#### :tophat: What? Why?

While reviewing a [bug report](https://meta.decidim.org/processes/bug-report/f/210/proposals/14971) I found that there's a problem on Consultations when there's a Question with Accountability component, where it gives an exception: 

> 
>  Showing /home/apereira/Work/decidim/decidim/decidim-consultations/app/views/layouts/decidim/_question_components.html.erb where line #7 raised:
> 
> Webpacker can't find media/images/decidim_accountability.svg in /home/apereira/Work/decidim/decidim/development_app/public/decidim-packs/manifest.json. Possible causes:
> 1. You want to set webpacker.yml value of compile to true for your environment
>    unless you are using the `webpack -w` or the webpack-dev-server.
> 2. webpack has not yet re-run to reflect updates.
> 3. You have misconfigured Webpacker's config/webpacker.yml file.
> 4. Your webpack configuration is not creating a manifest.
> Your manifest contains:
> (...)

This is because in the accountability component manifest we've a filename but in the app/packs/images/ we have another one, so the fix is an easy rename. 

https://github.com/decidim/decidim/blob/d6e49772e59ab898424c831026651ec1b3c296de/decidim-accountability/lib/decidim/accountability/component.rb?plain=1#L8

While reviewing this, I found out another bug related with the mobile navigation:

![image](https://user-images.githubusercontent.com/717367/162711355-dea58eb3-d1b2-4334-8b85-86db37bd7e1a.png)

There are two problems with this screenshot: 

1. There are two components navbars
2. It's showing the components icon, this has been dropped for other spaces a few years ago. 

So this PR fixes those problems too. 


#### :pushpin: Related Issues

- Related to https://meta.decidim.org/processes/bug-report/f/210/proposals/14971 

#### Testing

1. Sign in as admin 
2. Go to admin dashboard
3. Go to a question's consultations
4. Add a new Accountability component
5. See this component in frontend with the Mobile Responsive Mode of your browser

### :camera: Screenshots

(After this PR)

![image](https://user-images.githubusercontent.com/717367/162711722-676c410c-f02f-4822-919c-46575ef0d54c.png)


:hearts: Thank you!
